### DISCUSSION TITLE
blog-v2-1-with-table-of-contents-and-anchor-links/

### DISCUSSION THREAD

**Discussion by giscus** [JakeSteam/blog-programming#30] [Comments]
# blog-v2-1-with-table-of-contents-and-anchor-links/

Whilst Jekyll’s Minima theme is great, there’s a couple of very basic features I wish it had by default. I added the features myself today, and decided to start doing “release notes” style posts for the codebase used in both my blogs!

https://blog.jakelee.co.uk/blog-v2-1-with-table-of-contents-and-anchor-links/

**Comment by askemata** [comment]
Is there any way to show the TOC only on the pages that i choose?

**Reply by JakeSteam** [reply]
There's not currently a built-in way, BUT if you wanted to implement it yourself, you could [change post.html](https://github.com/JakeSteam/minimaJake/blob/main/_layouts/post.html#L25) and replace:

```
  {% if site.table_of_contents_header != blank? %}
```
with 
```
  {% if page.table_of_contents_header != blank? %}
```

This will make the table of contents appear on every page with `table_of_contents_header` defined in the front matter (where the title etc is). However the TOC won't have a header. This can be fixed easiest by hardcoding it [within contents.html](https://github.com/JakeSteam/minimaJake/blob/main/_includes/custom/contents.html#L173) to "Jump to:" or similar.

Hope that helped!
